configuration/online-telemetry.adoc

```diff
@@ -0,0 +1,54 @@
+[id="online-telemetry"]
+= Understanding online telemetry
+include::modules/common-attributes.adoc[]
+:context: online-telemetry
+
+toc::[]
+
+[role="_abstract"]
+{product-title} includes online telemetry, that is, it optionally gathers data about {product-title} and its components.
+The online telemetry data includes various operational and usage metrics, as well as information about the size, shape, and possible adverse conditions of the environments {product-title} components are running in.
+
+{product-title} does not collect node names, IP addresses, or other identifiable information about your infrastructure.
+See the xref:#collected-data[collected data] section for detailed information about the data {product-title} collects.
```

Comment:
This xref didn't work. You'll want to do the full file path `xref:../configuration/ ...` even though it's in the same file.